kivy/uix/screenmanager.py

```diff
@@ -909,8 +909,8 @@ def on_current(self, instance, value):
             self.transition.screen_out = previous_screen
             self.transition.start(self)
         else:
-            screen.pos = self.pos
```

Comment:
We still need to set the pos,  ScreenManager inherits from a FloatLayout which does not manage the position of it's children unless the child uses pos_hint.